USAGE.md

```diff
@@ -135,3 +135,13 @@ The replication throttler only affects the copy-rows operation, and does not app
 - Adjusting the configuration of your replicas to increase the parallel replication threads
 - Temporarily disabling durability on the replica (i.e. `SET GLOBAL sync_binlog=0` and `SET GLOBAL innodb_flush_log_at_trx_commit=0`)
 - Increasing the `replica-max-lag` or disabling replica lag checking temporarily
+
+### defer-cutover
+
+The "defer cutover" feature makes spirit wait to perform the final cutover until a "sentinel" table has been dropped. This is similar to the --postpone-cut-over-flag-file feature of gh-ost.
+
+If defer-cutover is true, Spirit will create a "sentinel" table in the same schema as the table being altered; the name of the sentinel table will use the pattern `_<table>_sentinel`. Spirit will never delete the sentinel table on its own. It will block for 48 hours waiting for the sentinel table to be dropped by the operator, after which it will exit with an error.
+
+You can resume a migration from checkpoint and Spirit will start waiting again for you to drop the sentinel table. You can also choose to delete the sentinel table before restarting Spirit, which will cause it to resume from checkpoint and complete the cutover without waiting, even if you have again enabled defer-cutover for the migration.
+
+If you start a migration and realize that you forgot to set defer-cutover, worry not! You can manually create a sentinel table using the pattern `_<table>_sentinel`, and Spirit will detect the table before the cutover is completed and block as though defer-cutover had been enabled from the beginning.
```

Comment:
docs LGTM, but please make it clear where the waiting occurs. Currently it is just before checksum, which has a valid use (checksum does require a lock to get started), but I would have thought just before cutover was better (I'll comment more on that inline).

If we decide defer-cutover happens before checksum:
- Document that it could still be several hours after the sentry table is dropped that the cutover actually occurs, when checksum is enabled (it is by default).

If we decide it's after checksum:
- Document that a lock is still required for checksum (could cause a brief stall)
- Cutover should be close to instant once the sentinel table is removed.

Reply:
Yes I can see both of them, but the fact we wait up to 48h for the sentinel table to be dropped, then exit and require a checkpoint resume, means that a checksum taken before the wait would have very little validity. If many users want to have the other behavior or more fine-grained control over both checksum *and* cutover, we can reassess in the future.